src/engine/safeReader.cpp

```diff
@@ -30,7 +30,7 @@ bool SafeReader::seek(ssize_t where, int whence) {
       curSeek=where;
       break;
     case SEEK_CUR: {
-      ssize_t finalSeek=len+where;
+      ssize_t finalSeek=curSeek+where;
```

Comment:
Reviewer FYI: I've already checked this wasn't used anywhere in the codebase - this PR is the first usage.

Reply:
Oops - good catch! I don't know why did I put len instead of curSeek...